### DISCUSSION TITLE
Tribes or sub-Families?

### DISCUSSION THREAD

**Discussion by schwartsburd** [pteridogroup/ppg#23] [Ideas]
As pointed out by the International Code of Nomenclature (Art. 4.1), the main and natural secondary rank between a Family and a Genus is a Tribe. On article 4.2, there is a further option to include third ranks, using the prefix "sub-". Therefore between Family and Genus, we have: Family, sub-Family, Tribe, sub-Tribe, Genus.
Note 1 of the Code says it is not mandatory to follow this classification.
I understand that sub-Family has been largely used in the Systematics of ferns, but not always. Tryon & Tryon (Ferns, allied pl. Tropical America, 1982), for example, adopted the use of Tribes instead of sub-Families.
Since we are using only one rank between Family and Genus, for me is more logical to adopt Tribe instead of sub-Family.
I also propose we use Tribes (or sub-Families) only to those bigger Families, such as Pteridaceae, Dryopteridaceae, Polypodiaceae, etc., similar to PPG I - even though I myself used sub-Families of Dennstaedtiaceae in a paper, for the sake of discussion, and not classification.
In addition, Tribes are easier speakable compared to sub-Families: Grammitideae versus Grammitidoideae; Polypodieae versus Polypodioideae.
I realize this proposal will be itching.

**Comment by joelnitta** [comment]
This seems to me to be solely an issue of semantics - do we call the rank below family "subfamily" or "tribe", and make the corresponding changes to the Latin names or not? Changing the names of taxa always comes at a cost of updating systems to match the new taxonomy. This may be warranted if it is needed to maintain a natural taxonomy (monophyletic taxa). But in this case, there is no new circumscription or actual change in taxonomic concept, so I don't think it is worth the effort just to have ranks that may be a bit more elegant.

**Comment by choess** [comment]
My intuitive reaction is like Joel's; I am not sure it is worth the disruption to existing literature to change the rank. I could be reasoned out of this, however.

I think it would be OK for us to entertain proposals for other infrafamilial groups, with reserve. I believe within the past few years I have seen tribes published for subdivisions of Microsoroideae. The principal considerations in proposing infrafamilial groups, for me, would be:

1. Is the clade phylogenetically robust and well-sampled? (That is, is the circumscription likely to be stable into the future?)
2. Are there sufficient synapomorphies for the clade to make this a useful named group? One would prefer to avoid monotypic tribes, subfamilies, etc. as well, when the phylogeny permits. I would set a reasonably high bar here; otherwise it is very tempting to put formal names onto clades that are not very useful in practice.

Using subfamilial rank for clades rooted close to the root of the family also has the advantage that, if the subfamily proves reasonably divisible (as in the Microsoroideae above), there is more room for subdivision without having to change ranks.
